test/cql-pytest/run.py

```diff
@@ -355,7 +355,7 @@ def run_pytest(pytest_dir, additional_parameters):
         run_pytest_pids = set()
         os.chdir(pytest_dir)
         os.setsid()
-        os.execvp('pytest', ['pytest',
+        os.execvp('../pytest', ['pytest',
```

Comment:
It was incredibly short-sighted of Fedora to add that silly "-s" to their "pytest" script - although they did it deliberately (see discussion in https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/ - the key sentence is "ensures the user’s Python packages (e.g. installed by pip install --user, or just placed in the current directory) don’t interfere with the RPM installed software." but they also say "Sometimes, such content is desirable, such as with plugins.".
I'll report this as a Fedora bug, but until then, this workaround is good.

However, I don't think that "../pytest" is right. Who said that pytest_dir (the current directory) is a direct subdirectory of the test/ directory? Instead in other places in this same file, we already have
```
source_path = os.path.realpath(os.path.join(__file__, '../../..'))
```

Because we know that this file is always $SCYLLADIR/test/cql-pytest/run.py, we know that `source_path` will point to the $SCYLLADIR always and can work from there. Then we don't need to assure where pytest_dir is.

Reply:
I opened https://bugzilla.redhat.com/show_bug.cgi?id=2152171 because I think this is a Fedora bug. But I agree we need to work around it until they fix it (if they ever do...).

Reply:
It didn't take them more than an hour to be convinced and send a patch that removes this silly "-s"... We should have done this long ago, when we noticed this problem even before Fedora 37 was released.

So, now, should we commit this workaround now - or wait for the yum repositories to be updated (I don't know how much time this will take...)

Reply:
I say commit now and revert (together with https://github.com/scylladb/scylladb/commit/e5e7780f32d20a328c8981ae80e2f080323482c4) as soon as Fedora 37 (and the dbuild container) is updated.
It is very frustrating for developers to try to run pytests with the run script and be greeted with the failing `cassandra` import. I have spent at least an hour trying to install various driver packages and browsing our run script and test.py until I realized what the problem is.

Reply:
Done in v2.

Reply:
Thanks. I'm on vacation today so I can't commit this PR, but I just noticed that Fedora's review process is faster than ours and supposedly the fix removing the wrong "-s" is already in :-)